src/lib/components/backoffice/Sidebar/Sidebar.js

```diff
@@ -60,15 +60,15 @@ class Sidebar extends Component {
                   <Menu.Header>Actions</Menu.Header>
                   <Menu.Menu>
                     <Menu.Item
-                      as={Link}
+                      as={checkInActive ? '' : Link}
```

Comment:
(suggestion)
What about defining an intermediary function for these? Or even better: a component that takes `to` and `active` props (+ `children`)?

Reply:
Nice work! @KonstantinaStoikou do you maybe want just to comment on this suggestion?

Reply:
I changed it to a function but because of [this](https://github.com/inveniosoftware/react-invenio-app-ils/pull/211#discussion_r498370273) comment I changed back to the initial component. IMO I don't think making a new component to wrap the `Menu.Item` offers any functionality.

Reply:
It was mainly to avoid repeating the same boolean variable twice and to factor the ternary expressions (all of that being typo prone). But it's not a big deal and I'm totally fine without it.